src/lib.rs

```diff
@@ -16,8 +16,15 @@
 
 #[macro_use]
 extern crate alloc;
+#[cfg_attr(feature = "fuzzing", macro_use)]
+extern crate arrayref;
+#[cfg_attr(feature = "fuzzing", macro_use)]
+extern crate cbor;
+#[cfg(not(feature = "fuzzing"))]
```

Comment:
Looks good, although we won't need these anymore when migrating to Rust 2018 edition (#118).

Reply:
Resolved by #156.